### PR TITLE
TestCase for forms

### PR DESCRIPTION
**Pull request by joe513**
Added some more tests for the coverage increasing

### CODE REVIEW

**Comment by joe513** [comment]
inspired by [answer](https://stackoverflow.com/a/26136993)

File: app/tests.py [hunk 1]
```diff
@@ -80,6 +78,19 @@ def test_channel_update(self):
         self.assertEqual(response.status_code, 302)
         self.assertIn('login', response.url, msg='Not redirected to login view')
 
+    def test_form(self):
+
+        add_playlist = self.client.post(reverse('create-playlist'), data={'path': 'https://dailyiptvlist.com/dl/fr-m3uplaylist-2018-03-06.m3u'})
+        self.assertEqual(add_playlist.status_code, 200)
+
+        new_channel = self.client.post(reverse('new-channel'), data={'path': 'https://archive.org/download/fluxustv/Fluxus_TV.mp4', 'title': 'Simple',
```
Comment (inoks) [inline app/tests.py]: @joe513 You can use https://pypi.python.org/pypi/requests-mock here to emulate loading data from remote server.

Comment (inoks) [reply app/tests.py]: @joe513 For example, we can test loading remote playlist by creating mocked response for custom url, like `http://example.com/playlist.m3u8`.  Please check docs: http://requests-mock.readthedocs.io/en/latest/response.html#registering-responses

Comment (joe513) [reply app/tests.py]: @inoks thank you very much

**Pull request closed without merging**